deeprankcore/utils/buildgraph.py

```diff
@@ -36,24 +37,74 @@ def _add_atom_to_residue(atom, residue):
     residue.add_atom(atom)
 
 
-def get_structure(pdb, id_): # pylint: disable=too-many-locals
+_amino_acids_by_code = {
+    amino_acid.three_letter_code: amino_acid for amino_acid in amino_acids
+}
+
+
+_elements_by_name = {element.name: element for element in AtomicElement}
+
+
+def _add_atom_data_to_structure(structure: PDBStructure,  # pylint: disable=too-many-arguments, too-many-locals
+                                x: float, y: float, z: float,
+                                atom_name: str,
+                                altloc: str, occupancy: float,
+                                element_name: str,
+                                chain_id: str,
+                                residue_number: int,
+                                residue_name: str,
+                                insertion_code: str):
```

Comment:
Can you add a docstring for this.
Also, is there a reason that altloc and occupancy are on the same line?